contingency/contingency-dsl/src/main/groovy/com/powsybl/contingency/dsl/ContingencyDslLoader.groovy

```diff
@@ -44,11 +46,18 @@ class ContingencyDslLoader extends DslLoader {
     }
 
     static void loadDsl(Binding binding, Network network, Consumer<Contingency> consumer, ContingencyDslObserver observer) {
-
         // contingencies
         binding.contingency = { String id, Closure<Void> closure ->
             def cloned = closure.clone()
             ContingencySpec contingencySpec = new ContingencySpec()
+
+            List<Extension<Contingency>> extensionList = new ArrayList<>();
+            for (ExtendableDslExtension dslContingencyExtension : ServiceLoader.load(ExtendableDslExtension.class)) {
+                if (dslContingencyExtension.getExtendableClass().equals(Contingency.class)) {
```

Comment:
We should introduce a ContingencyDslExtension interface to make this code more compact. The generic seems useless.